sdk/servicebus/service-bus/review/service-bus.api.md

```diff
@@ -302,31 +322,63 @@ export class ServiceBusMessage implements ReceivedMessage {
     viaPartitionKey?: string;
 }
 
+// @public
+export interface ServiceBusReceiverClient {
+    new (queueAuth: QueueAuth, receiveMode: "peekLock"): ClientTypeT<"peekLock", "queue", "nosessions">;
+    new (queueAuth: QueueAuth, receiveMode: "receiveAndDelete"): ClientTypeT<"receiveAndDelete", "queue", "nosessions">;
+    new (queueAuth: QueueAuth, receiveMode: "peekLock", session: Session): ClientTypeT<"peekLock", "queue", "sessions">;
+    new (queueAuths: QueueAuth, receiveMode: "receiveAndDelete", session: Session): ClientTypeT<"receiveAndDelete", "queue", "sessions">;
```

Comment:
Nit (I realize this code is generated):
```suggestion
    new (queueAuth: QueueAuth, receiveMode: "receiveAndDelete", session: Session): ClientTypeT<"receiveAndDelete", "queue", "sessions">;
```

Reply:
No reason not to fix it. It's a Dwigt moment (I replaced all the others)